Resources/Prototypes/_EinsteinEngines/Entities/Mobs/NPCs/animals.yml

```
@@ -96,8 +122,8 @@
     visuals:
       enum.HarpyVisualLayers.Singing:
         singingLayer:
-          False: { visible: false }
-          True: { visible: true }
+          False:  {visible: false}
```

YAMLlint (1.37.1)

[warning] too many spaces after colon

(colons)